pkg/config/config.go

```diff
@@ -542,128 +541,26 @@ func shouldExcludeModule(ctx context.Context, root *pathlib.Path, goModPath *pat
 	return false, nil
 }
 
-func (c *Config) subPackages(
-	ctx context.Context,
-	pkgPath string,
-	pkgConfig *Config,
-	currentDepth int,
-) ([]string, error) {
-	log := zerolog.Ctx(ctx)
-
+func (c *Config) subPackages(pkgPath string) ([]string, error) {
 	pkgs, err := packages.Load(&packages.Config{
 		Mode: packages.NeedName | packages.NeedFiles,
-	}, pkgPath)
+	}, pkgPath+"...")
```

Comment:
Needs to be `"/..."` I believe.

Reply:
I did this, and also tested that it still works for my usecases.